### DISCUSSION TITLE
so-rule disabled not Ok

### DISCUSSION THREAD

**Discussion by elenezet** [Security-Onion-Solutions/securityonion#8490] [Unsupported Versions] [answered]
Hello,
I've a problem with rule sid 2019401
I user sudo so-rul disabled add 2019401, this rule is in /opt/so/saltstack/local/pillar/minions/XXX_managersearch.sls
When i list disabled rule it's ok ..
sudo so-rule disabled list
Disabled rules:
  - 2019401
But it was not disabled in all.rules :

grep 2019401 all.rules 
alert http $HOME_NET any -> $EXTERNAL_NET any (msg:"ET POLICY Vulnerable Java Version 1.8.x Detected"; flow:established,to_server; flowbits:set,ET.http.javaclient.vulnerable; threshold: type limit, count 2, seconds 300, track by_src; http.user_agent; content:"Java/1.8.0_"; content:!"291"; within:3; reference:url,www.oracle.com/technetwork/java/javase/8u-relnotes-2225394.html; classtype:bad-unknown; sid:2019401; rev:35; metadata:affected_product Java, attack_target Client_Endpoint, created_at 2014_10_15, deployment Perimeter, deployment Internal, former_category POLICY, performance_impact Low, signature_severity Informational, updated_at 2021_12_22;)
It work for another rules ... but for this one i can't disabled it :( 
any idear?
Thanks

**Comment by xfaith** [comment]
Can you show your code on how you set up your disabled rule?  
Per the guide, it should look like this.  
(Remember to watch spacing.  2 spaces for every indent.  Also mind the colons : )

idstools:
  sids:
    disabled:
      - 4321

**Comment by elenezet** [comment]
disabled:
    - '2019401'
I found this https://redmine.openinfosecfoundation.org/issues/3102 but i don't understant what i must do to disable this rule .. any idear?
I search ...

**Comment by dougburks** [comment, marked as answer]
Please see the Flowbits section of the documentation:
https://docs.securityonion.net/en/2.3/managing-alerts.html#flowbits

**Comment by elenezet** [comment]
it works now with flowbits options. Thanks.
